#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-444648.dpatch by Bernhard R. Link <brlink@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do proper PS stream encoding on 64 bit architectures

@DPATCH@
--- a/xpdf/Stream.cc	2008-09-04 11:57:18.000000000 +0000
+++ b/xpdf/Stream.cc	2008-09-04 12:33:13.000000000 +0000
@@ -4514,7 +4514,7 @@
 GBool ASCII85Encoder::fillBuf() {
   Gulong t;
   char buf1[5];
-  int c0, c1, c2, c3;
+  unsigned int c0, c1, c2, c3;
   int n, i;
 
   if (eof) {
